tests/e2e/hcp_tuning_config_test.go

```diff
@@ -0,0 +1,111 @@
+package e2e
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/openshift/rosa/tests/ci/labels"
+	"github.com/openshift/rosa/tests/utils/common"
+	"github.com/openshift/rosa/tests/utils/exec/rosacli"
+)
+
+var _ = Describe("Create Tuning Config", func() {
+
+	var rosaClient *rosacli.Client
+	var clusterService rosacli.ClusterService
+
+	BeforeEach(func() {
+		Expect(clusterID).ToNot(BeEmpty(), "Cluster ID is empty, please export the env variable CLUSTER_ID")
+		rosaClient = rosacli.NewClient()
+		clusterService = rosaClient.Cluster
+
+		By("Check hosted cluster")
+		hosted, err := clusterService.IsHostedCPCluster(clusterID)
+		Expect(err).ToNot(HaveOccurred())
+		if !hosted {
+			Skip("Tuning Configs are only supported on Hosted clusters")
+		}
+	})
+
+	AfterEach(func() {
+		By("Clean remaining resources")
+		err := rosaClient.CleanResources(clusterID)
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	It("tuning config can be created/updated/deleted to hosted cluster - [id:63164]",
+		labels.Critical, labels.NonClassicCluster,
+		func() {
+			tuningConfigService := rosaClient.TuningConfig
+			tuningConfigName_1 := common.GenerateRandomName("tuned01", 2)
+			tuningConfigName_2 := common.GenerateRandomName("tuned02", 2)
+
+			tuningConfigPayload_1 := `{
+			"profile": [
+				{
+				"data": "[main]\nsummary=Custom OpenShift profile\ninclude=openshift-node\n\n[sysctl]\nvm.dirty_ratio=\"25\"\n",
+				"name": "%s-profile"
+				}
+			],
+			"recommend": [
+				{
+				"priority": 10,
+				"profile": "%s-profile"
+				}
+			]
+		}`
+			tuningConfigPayload_2 := `{
+			"profile": [
+				{
+					"data": "[main]\nsummary=Custom OpenShift profile\ninclude=openshift-node\n\n[sysctl]\nvm.dirty_ratio=\"65\"\n",
+					"name": "%s-profile"
+				}
+			],
+			"recommend": [
+				{
+					"priority": 20,
+					"profile": "%s-profile"
+				}
+			]
+		}`
+			By("Create tuning configs to the cluster")
+			_, err := tuningConfigService.CreateTuningConfig(clusterID, tuningConfigName_1, fmt.Sprintf(tuningConfigPayload_1, tuningConfigName_1, tuningConfigName_1))
+			Expect(err).ToNot(HaveOccurred())
```

Comment:
It should check the creation response too

Reply:
I updated it. Thanks for it